charts/values.schema.json

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
It is up to this repository maintainers to accept this validation piece. I believe it should be plugged into CI in order to make sense.

Reply:
Thanks for that feedback @slaskawi